### PR TITLE
Update XCM Tutorials for XCM V3

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the Remote Staking and Uniswap V2 via XCM docs to use XCM V3.

⭕️ Once this goes through an initial review, I can circle back and address the Uniswap V2 Swap images, if we want to keep them? The multilocation address has changed to v3 and the encoded call data has changed, so all of those images are outdated. ⭕️

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [x] Will need to create PR in `moonbeam-docs` repo to remove images

#### Items to be Updated

Some images from the Remote Staking guide are no longer used

### CODE REVIEW

File: .snippets/code/tutorials/interoperability/remote-staking/remote-staking.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  const api = await ApiPromise.create({ provider: wsProvider });
+
+  // 3. Initialize wallet key pairs
+  const keyring = new Keyring({ type: 'sr25519' });
+  // For demo purposes only. Never store your private key or mnemonic in a JavaScript file
+  const otherPair = keyring.addFromUri('YOUR-DEV-SEED-PHRASE-HERE');
+  console.log(`Derived Address from Private Key: ${otherPair.address}`);
+
+  // 4. Define the transaction using the send method of the xcm pallet
+  const tx = api.tx.xcmPallet.send(dest, message);
```
Comment (albertov19) [inline .snippets/code/tutorials/interoperability/remote-staking/remote-staking.js:55]:
```suggestion
  const tx = api.tx.xcmPallet.send(xcmDest, xcmMessage);
```

File: .snippets/code/tutorials/interoperability/remote-staking/remote-staking.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { ApiPromise, WsProvider, Keyring } from '@polkadot/api';
+
+// 1. Define the dest and message arguments
+const dest = { V3: { parents: 0, interior: { X1: { Parachain: 1000 } } } };
```
[the remark this answers is not in the document]
Comment (eshaben) [reply .snippets/code/tutorials/interoperability/remote-staking/remote-staking.js:4]: our other xcm pages just use `dest` and `message` - i think we should leave it as is for consistency

[the remark this answers is not in the document]
Comment (eshaben) [reply .snippets/code/tutorials/interoperability/remote-staking/remote-staking.js:6]: ah I see... in all the tutorials I updated I don't append `xcm` to the variables but I see it's this way in the Uniswap tutorial

**Comment by eshaben** [comment]
ok so I went through all of the XCM-related guides and tutorials and made the following changes for consistency:

- use `xcmDest` and `xcmMessage` in favor of `dest` and `message` for the actual values (this doesn't apply to function parameter names, which are `dest` and `message`)
- use `PARAMETER_NAME` instead of `PARAMETER-NAME`
- use format of `INSERT_PARAMETER_NAME` instead of `PARAMETER_NAME_HERE` or `INSERT_PARAMETER_NAME_HERE`
- only use capitalization of parameters when it refers to exported constants (which we don't ever really use, so we shouldn't ever capitalize parameters)
- use snake_case for all variables, functions, etc., in python snippets

✅ Will make sure that these formatting changes are documented in the style guide
✅ Will also create a ticket to go through the rest of the docs to make these updates (just time consuming and out of scope for this PR)

**Comment by eshaben** [comment]
Actually on second thought, I think it is easier to stick to a rule of using the same name as the parameter for the argument rather than having one-off rules like use `xcmDest` for `dest`. Then we can apply this rule across the board.

**Review by albertov19 — APPROVED** [review]
LGTM! Agree on your suggestion of using the parameter name as the variable name!

**Pull request merged**